### ISSUE TITLE
Still no entry for Node 33.0.2

### ISSUE THREAD

**Issue by cseickel** [electron/node-abi#195]
This is a repeat of #189. It was closed as fixed but no fix was actually made that I can see. There has also been no new releases since that issue was opened. If code was checked in that should fix it, maybe it just needs to be released?

**Comment by LordTocs** [comment]
This is still an issue for me, the latest node-abi still doesn't work with 33.0.2. Is the electron-node-abi-updater[bot] working? I don't see a commit with an update to the registry?

**Comment by cseickel** [comment]
@VerteDinde it doesn't seem like the fix you tried to make for #189 had any effect.

**Comment by dsanders11** [comment]
@cseickel, I can't reproduce your issue, it seems to work fine for Electron v33.0.2.

```
> const nodeAbi = require('node-abi')
> nodeAbi.getAbi('33.0.12, 'electron')
'130'
```

**Comment by cseickel** [comment]
My apologies @dsanders11, when I dug deeper I realized that I had an older version of node-abi. 3.60 was the version implicitly used as a dependency of electron forge 7.5.1. When I explicitly installed node-abi and got version 3.71.0, then I got the correct answer when looking up Electron v33.0.2.

Explicitly installing node-abi does not solve my problem though. The real issue here is that electron-forge uses rebuild@3.7.1, which uses node-abi@3.60.0, which is the true cause of my problem. 3.7.1 is the latest version of rebuild, so I guess that means electron forge is just not compatible with the 33.x series of electron releases yet.

All of this is not a fault of this project, so I will close the issue. Hopefully the above will help others troubleshoot in the future.

**Comment by cseickel** [comment]
Thanks for the info @MarshallOfSound. 

I use yarn for the project with the issue and `yarn upgrade node-abi` does not actually do anything, it will stay with 3.60.0. Explicitly installing node-abi will just give me an additioonal version but does not affect what rebuild uses. 

What worked for me was deleting `node-modules/` and `yarn.lock` and re-installing. On a fresh install, I got the latest version of node-abi and only the latest version. I assume `3.60.0` was just the latest version the last time I did a fresh install.
